ibis/backends/dask/__init__.py

```diff
@@ -10,3 +20,63 @@
         'See ibis.dask.trace for details.',
         validator=ibis.config.is_bool,
     )
+
+
+def connect(dictionary):
+    """Construct a dask client from a dictionary of DataFrames.
+
+    Parameters
+    ----------
+    dictionary : dict
```

Comment:
Can you please explain is the the `dictionary ` here? Also feels like this could be a bit better named

Reply:
this is already present in all other backends (agreed its not great)